scripts/generate_errors.pl

```diff
@@ -3,23 +3,22 @@
 # Generate error.c
 #
 # Usage: ./generate_errors.pl or scripts/generate_errors.pl without arguments,
-# or generate_errors.pl include_dir data_dir error_file
+# or generate_errors.pl include_dir data_dir error_c [error_h]
```

Comment:
Can you also update the `Generate error.c` line above?

And, while this is unrelated, we should probably also document that this script checks that no error code is duplicated (I was never sure about that).